Source/WebCore/Modules/webcodecs/WebCodecsAudioDecoder.cpp

```diff
@@ -63,9 +63,27 @@ WebCodecsAudioDecoder::~WebCodecsAudioDecoder()
 {
 }
 
-static bool isValidDecoderConfig(const WebCodecsAudioDecoderConfig& config, const Settings::Values&)
+static bool isValidDecoderConfig(const WebCodecsAudioDecoderConfig& config)
 {
-    return AudioDecoder::isCodecSupported(config.codec);
+    if (StringView(config.codec).trim(isASCIIWhitespace<UChar>).isEmpty())
+        return false;
+
+    if (!config.numberOfChannels)
+        return false;
+
+    if (!*config.numberOfChannels)
+        return false;
```

Comment:
I do not see this check and some below in https://w3c.github.io/webcodecs/#valid-audiodecoderconfig.
Where is it coming from?

Reply:
Well the WPT tests check that, and I see blink does too. Haven't checked gecko though.

Reply:
I'll revert this part and add a FIXME.